airflow-core/src/airflow/ui/public/i18n/locales/he/admin.json

```diff
@@ -81,6 +84,7 @@
     },
     "importError_one": "שגיאה בייבוא תוסף",
     "importError_other": "שגיאות בייבוא תוסף",
+    "importError_two": "שגיאות בייבוא תוסף",
```

Comment:
Are these supposed to refer to more than one issue at a time? If so, why not תוספים?

Reply:
```
    "importError_one": "Plugin Import Error",
    "importError_other": "Plugin Import Errors",
```

The English source refers to multiple issues in a single plugin